aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -68,25 +68,27 @@ HTTP/2 is disabled by default. For more information on configuration, see the [K
 
 ## When to use Kestrel with a reverse proxy
 
-You can use Kestrel by itself or with a *reverse proxy server*, such as IIS, Nginx, or Apache. A reverse proxy server receives HTTP requests from the Internet and forwards them to Kestrel after some preliminary handling.
+You can use Kestrel by itself or with a *reverse proxy server*, such as [Internet Information Services (IIS)](https://www.iis.net/), [Nginx](http://nginx.org), or [Apache](https://httpd.apache.org/). A reverse proxy server receives HTTP requests from the network and forwards them to Kestrel.
 
 ![Kestrel communicates directly with the Internet without a reverse proxy server](kestrel/_static/kestrel-to-internet2.png)
 
 ![Kestrel communicates indirectly with the Internet through a reverse proxy server, such as IIS, Nginx, or Apache](kestrel/_static/kestrel-to-internet.png)
 
-Either configuration&mdash;with or without a reverse proxy server&mdash;is a valid and supported hosting configuration for ASP.NET Core 2.0 or later apps.
+Either configuration&mdash;with or without a reverse proxy server&mdash;is a valid supported hosting configuration for ASP.NET Core 2.0 or later apps that receive requests from the Internet.
 
-A reverse proxy scenario exists when there are multiple apps that share the same IP and port running on a single server. Kestrel doesn't support this scenario because Kestrel doesn't support sharing the same IP and port among multiple processes. When Kestrel is configured to listen on a port, Kestrel handles all of the traffic for that port regardless of requests' host header. A reverse proxy that can share ports has the ability to forward requests to Kestrel on a unique IP and port.
+A reverse proxy scenario exists when there are multiple apps that share the same IP and port running on a single server. Kestrel doesn't support this scenario because Kestrel doesn't support sharing the same IP and port among multiple processes. When Kestrel is configured to listen on a port, Kestrel handles all of the traffic for that port regardless of requests' `Host` header. A reverse proxy that can share ports has the ability to forward requests to Kestrel on a unique IP and port.
```

Comment:
Avoid the double negative in the 2nd sentence.

Reply:
I think it's blown for a different reason. In this case, the 2nd negative doesn't describe (or negate) the action of the first ... it's in a different clause and pertains to the subject there.

Subject negative verb (action) **because** subject negative verb (condition).

> I don't eat strawberries because strawberries don't taste good.

However, the 2nd sentence has redundant thoughts ... and is too long ... let's go with .......

> Kestrel doesn't support sharing the same IP and port among multiple processes.

... or how about just ...

> Kestrel doesn't taste good.

.... wait!! *what??!!* :smile: lol